test/csuite/timestamp_abort/main.c

```diff
@@ -206,26 +277,15 @@ thread_run(void *arg)
 	    uri_oplog, NULL, NULL, &cur_oplog)) != 0)
 		testutil_die(ret, "WT_SESSION.open_cursor: %s", uri_oplog);
 
-	if ((ret = session->open_cursor(
-	    session, stable_store, NULL, NULL, &cur_stable)) != 0)
-		testutil_die(ret, "WT_SESSION.open_cursor: %s", stable_store);
-
 	/*
 	 * Write our portion of the key space until we're killed.
 	 */
-	printf("Thread %" PRIu32 " starts at %" PRIu64 "\n", td->id, td->start);
+	printf("Thread %" PRIu32 " starts at %" PRIu64 "\n",
+	    td->info, td->start);
 	for (i = td->start; ; ++i) {
-		if (use_ts) {
-			/*
-			 * There can be multiple threads doing transactions
-			 * simultaneously requiring us to do some co-ordination
-			 * so that a thread doesn't try to commit with a
-			 * timestamp older than the oldest_timestamp just bumped
-			 * by another thread.
-			 */
-			testutil_check(pthread_rwlock_rdlock(&commit_ts_lock));
+		if (use_ts)
 			stable_ts = __wt_atomic_addv64(&global_ts, 1);
-		} else
+		else
 			stable_ts = 0;
```

Comment:
I'm sure it doesn't matter, but why do `stable_ts = 0` on every step through the loop, set it once outside the loop.

I'd move the initialization of `stable_ts` down to where it's used now that locking isn't needed.